.devcontainer/Dockerfile

```diff
@@ -15,6 +15,6 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 RUN gem install foreman
 
 # [Optional] Uncomment this line to install global node packages.
-RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && npm install -g yarn" 2>&1
+RUN su vscode -c "source /usr/local/share/nvm/nvm.sh && corepack enable" 2>&1
```

Comment:
corepack shouldn't be needed if going to NPM

Reply:
It depends if we want to lock the `npm` version. I am not sure what is the best practice here.